docs/releases/overview.md

```diff
@@ -7,23 +7,86 @@ The Vega software is publicly available on [GitHub](https://github.com/vegaproto
+
+- The `vega node` command has been renamed to `vega start`. 
+- The `vega tm` command has been renamed to `vega tendermint`. 
+- The `Blockchain.Tendermint.ClientAddr` configuration field has been renamed to `Blockchain.Tendermint.RPCAddr`. 
+- The `init` command now also generates the configuration for tendermint, and also has the newly introduced flags `--no-tendermint`,` --tendermint-home` and `--tendermint-key`.
+This work was all done in issue [5579](https://github.com/vegaprotocol/vega/issues/5579)
+
+**Remove `updateFrequency` in price monitoring definition:**
+The `updateFrequency` within price monitoring is not being used by the core protocol, therefore, this has now been replaced by the update frequency of the state var. This work was done in issue [5624](https://github.com/vegaprotocol/vega/issues/5624)
+
+**Remove wallet support for launching a proxy in front of dApps:**
+Introducing the proxy was a way to navigate the browser security that prevents webapps from being able to talk to local web servers; this is now no longer required and therefore has been removed. This has been carried out under the issue [5601](https://github.com/vegaprotocol/vega/issues/5601)
+
+**Critical Bug Fixes**
+
+**Updating oracle termination causes the core to panic:**
+During a recent incentive on testnet it was found that user updating the oracle termination conditions on an enacted continuous market caused the network to panic. The oracle termination condition was set to be ten seconds in the future; when the network reached this time the core threw a panic.
+This bug has been resolved in [5668](https://github.com/vegaprotocol/vega/pull/5668)
+
+**Network parameter set to `0` can cause node startup failure:**
+An issue was discovered using the [Market Simulator](https://github.com/vegaprotocol/vega-market-sim) when the governance parameter `governance.proposal.updateMarket.minProposerEquityLikeShare` is set to `0` in the `genesis.json`, this resulted in the node startup failing. The fix implemented in [5633](https://github.com/vegaprotocol/vega/issues/5633) addresses this and allows the value to be `0`.
+
+**Cannot unregister order causing core to crash in Market Simulator:**
+When using the [Market Simulator](https://github.com/vegaprotocol/vega-market-sim) it was found that a "cannot unregister order" error was thrown and caused the core to crash. This bug has been resolved in [5663](https://github.com/vegaprotocol/vega/issues/5663)
+
+**Entering an auction caused by pegged order causes the core to panic:**
+It was identified that during a markets life when it is not in an auction and a pegged order gets repriced the order was removed from the book, however, it remained in the pegged list. If the market then goes into an auction the pegged order list is used to try to get the order from the book, however in this case the order no longer exists, causing a panic. This bug has been resolved in [5825](https://github.com/vegaprotocol/vega/issues/5825)
```

Comment:
```suggestion
It was identified that during a market's life when it is not in an auction and a pegged order gets repriced, the order was removed from the book, however, it remained in the pegged orders list. If the market then goes into an auction the pegged order list is used to try to get the order from the book, but as the order no longer exists, it can't be done. This bug has been resolved in [5825](https://github.com/vegaprotocol/vega/issues/5825)
```